stubs/types/container.stub

```diff
@@ -0,0 +1,10 @@
+{{{
+  exports({ to: app.makePath('types/container.ts') })
+}}}
+declare module '@adonisjs/core/types' {
+  // import ExampleJob from '#jobs/example_job'
```

Comment:
When not commented-out, Typescript complains about the following:

`TS2667: Imports are not permitted in module augmentations. Consider moving them to the enclosing external module.`

and 

`TS2307: Cannot find module #jobs/test_job or its corresponding type declarations.`


It looks like for the first one maybe this is related? https://github.com/microsoft/TypeScript/pull/57704